src/cli/cmds/fleet.toit

```diff
@@ -324,7 +324,8 @@ create-fleet-commands config/Config cache/Cache ui/Ui -> List:
       --run=:: group-list it config cache ui
   group-cmd.add group-list-cmd
 
-  group-create-cmd := cli.Command "create"
+  group-add-cmd := cli.Command "add"
+      --aliases=["create"]
       --help="""
         Create a group in the fleet.
```

Comment:
Not sure if you want to update the descriptions too.

Reply:
done.